.circleci/config.yml

```diff
@@ -566,7 +578,19 @@ workflows:
             tags:
               only: /v\d+\.\d+\.\d+(-[\w]+)?/
       - integration_test:
-          name: bigquery-itest
+          name: bigquery-no-gcs-itest
+          extras: '[bigquery,itest]'
+          python_version: "3.6"
+          db_name: bltoolsdevbq-bq_itest
+          include_gcs_scratch_bucket: false
+          requires:
+            - redshift-s3-itest
+          filters:
+            tags:
+              only: /v\d+\.\d+\.\d+(-[\w]+)?/
+      - integration_test:
+          # TODO: Get service account set in this one
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker